packages/@n8n_io/types/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@n8n_io/types",
```

Comment:
this will have to wait until we have the new release process in place, as the current release process needs to be manually updated for every additional package.

Reply:
this doesn't have to be compiled, it's just `types`

Reply:
I mean I think it is safe there even if nothing happens to it. It's just a common place to import common types from (but I can be wrong)